paymentsheet/src/main/java/com/stripe/android/link/ui/inline/LinkInlineSignup.kt

```diff
@@ -131,34 +134,17 @@ internal fun LinkInlineSignup(
     val scope = rememberCoroutineScope()
     val emailFocusRequester = remember { FocusRequester() }
     val bringFullSignUpIntoViewRequester = remember { BringIntoViewRequester() }
+    val simplifiedCheckbox = linkSignUpOptInFeatureEnabled || allowsDefaultOptIn
```

Comment:
DOI and the toggle share the same wrapping UI (borderless etc). the toggle just does not show collection fields at all.